lib/utils/conf.py

```diff
@@ -21,6 +20,7 @@
 import toml
 import copy
 from jsonschema import validate
+from typing import Mapping
```

Comment:
It looks like most of typing was deprecated in Python 3.9. https://docs.python.org/3/library/typing.html#typing.Mapping I think the preferred method is `from collections.abc import Mapping` which will fail for anything older than Python 3.9 So how about something like...
```
try:
    from collections.abc import Mapping
except ImportError:
    from collections import Mapping
```

Reply:
Ah, did not realize.  We only use Python 3.10